### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by Jay-Gadhiya**
card component contains simple card, with badge, with dismiss button and shadows

### CODE REVIEW

**Comment by mohammed-sas** [comment]
suggestion : add screenshot while raising pr, it would be easier to review

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
suggestion : ask captain if it's okay to make card component with different html structure as your variants had different structure . 
card with shadow : there were three cards in in this variant , display one card.

**Comment by gadgetfather** [comment]
image
Your "add to cart" is small compared to the "add to wishlist" button. 
rest is good 👍

**Review by gadgetfather — APPROVED** [review]

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    padding-right: .7rem;
+    cursor: pointer;
+}
+
+.icon-heart {
+    color: gray;
+}
+
+.icon-heart.heart-icon-color {                 /*toogle using js*/
+    color: #f44336;
```
Comment (gopalbbharadva) [inline components/card/card.css]: Suggestion: Use variables instead of hard coded color value.

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+}
+
+.badge-new {
+    position: absolute;
+    top: 15px;
+    left: 0;
+    background-color: #e66da9;
+    padding: .3rem 1.5rem;
+    border-radius:5px;
+    color: #fff;
```
Comment (gopalbbharadva) [inline components/card/card.css]: Suggestion: use CSS variables.

**Review by gopalbbharadva — APPROVED** [review]
Your PR request should have only files related to that PR only.(E.g. if your PR is about card component then your PR should not contain any other file changes.)

**Pull request merged**